File: docs/examples/python-langchain-fastapi/handlers/basic_handler.py
from ygo74.agent_runtime.domains.contracts.exchange_models import StandardExchangeRequest, StandardExchangeResponse


def basic_handler(request: StandardExchangeRequest) -> StandardExchangeResponse:
    return StandardExchangeResponse(request_id=request.request_id, status="success", output={"message": "ok"})

File: docs/examples/java-springai-springboot/src/main/java/com/ygo74/examples/BasicHandler.java
package com.ygo74.examples;

import com.ygo74.agentruntime.domains.contracts.ExchangeModels.StandardExchangeRequest;
import com.ygo74.agentruntime.domains.contracts.ExchangeModels.StandardExchangeResponse;

import java.util.Map;

public class BasicHandler {
    public StandardExchangeResponse handle(StandardExchangeRequest request) {
        return new StandardExchangeResponse(request.requestId(), "success", Map.of("message", "ok"), null, Map.of());
    }
}
